daemon/container_operations_windows.go

```diff
@@ -58,3 +43,7 @@ func isLinkable(child *container.Container) bool {
 func enableIPOnPredefinedNetwork() bool {
 	return true
 }
+
+func (daemon *Daemon) supportsHotAdd() bool {
```

Comment:
@msabansal the word `Hot` gives the context for `Pluggable` :-) without that, it is easy to confuse it with Plugins....